api/src/permissions/modules/validate-remaining-admin/validate-remaining-admin-count.ts

```diff
@@ -0,0 +1,9 @@
+import { UnprocessableContentError } from '@directus/errors';
+
+export function validateRemainingAdminCount(count: number) {
+	if (count <= 0) {
+		throw new UnprocessableContentError({
+			reason: `Doing this would remove the last admin user from the system`,
```

Comment:
"Doing this" sounds a bit too generic IMO 😇 
```suggestion
			reason: `Cannot remove the last admin user from the system`,
```